.github/workflows/lighthouse-audit.yml

```diff
@@ -0,0 +1,79 @@
+name: Lighthouse audit
```

Comment:
If we need to add another version of this for paypal fastlane, do we need to duplicate this file?

Reply:
Yes, we will need to either duplicate or modify it to test against the PayPal setup.